lib/pom.xml

```diff
@@ -1,22 +1,25 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-        xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-        <modelVersion>4.0.0</modelVersion>
-        <groupId>pro.taskana</groupId>
-        <artifactId>taskana-parent</artifactId>
-        <version>1.1.3-SNAPSHOT</version>
-        <packaging>pom</packaging>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<artifactId>taskana-lib-parent</artifactId>
+	<version>1.1.3-SNAPSHOT</version>
```

Comment:
Eclipse recommends to have the verion in the parent only. This is just duplicate information. And I agree.

Reply:
Yes and no.
It is duplicate information, but since the release modifies the version the duplicate information is necessary. Otherwise it throws an error because the version is inherited from the parent.
See: https://travis-ci.org/mustaphazorgati/taskana/jobs/530279994

Reply:
My understanding would be that we have to change the version on the parent, only, and the child modules are updated automatically. This is not true?

See https://stackoverflow.com/questions/5726291/updating-version-numbers-of-modules-in-a-multi-module-maven-project

Reply:
That is true. But since we are deploying the child modules by themself the version tag is necessesary because the mvn deploy command tries to change the version.